src/controllers/token.rs

```diff
@@ -110,3 +110,21 @@ pub fn revoke(req: &mut dyn RequestExt) -> EndpointResult {
     struct R {}
     Ok(req.json(&R {}))
 }
+
+/// Handles the `DELETE /tokens/current` route.
+pub fn revoke_current(req: &mut dyn RequestExt) -> EndpointResult {
+    let authenticated_user = req.authenticate()?;
+    let api_token_id = authenticated_user
+        .api_token_id()
+        .ok_or_else(|| bad_request("token not provided"))?;
+    let conn = req.db_conn()?;
+    diesel::delete({
+        use self::api_tokens::dsl::*;
+        api_tokens.filter(id.eq(api_token_id))
+    })
```

Comment:
@mibac138 was there a reason why you chose to delete the token instead of setting `revoked: true` like the `revoke()` function above does it? It would probably be best to use the same behavior for both implementations unless there are strong reasons against it.

Reply:
I think I just did that by mistake, fixed now